modules/perception/obstacle/camera/detector/yolo_camera_detector/yolo_camera_detector.cc

```diff
@@ -381,10 +381,13 @@ bool YoloCameraDetector::Detect(const cv::Mat &frame,
   for (auto &obj : *objects) {
     projector_->project(&(obj->object_feature));
 
-    // Assign unique detection box id per box for this frame
-    obj->id = det_id;
+    // Assign detection id per box and score
+    obj->id = det_id++;
+    for (auto prob : obj->type_probs) {
+      obj->score = std::max(obj->score, prob);
+    }
+
     AINFO << "obj-" << det_id << ": " << obj->object_feature.size();
```

Comment:
I would suggest using the original method to increase det_id. Otherwise, at line 390, the det_id is not the id for obj, and the AINFO will be misleading.

Reply:
Fixed